### ISSUE TITLE
gene SVG glyph

### ISSUE THREAD

**Issue by rbuels** [GMOD/jbrowse-components#241]
basically need to port the gene glyph from JB1, but with svg

**Comment by cmdcolin** [comment]
Possible add-on: semantic zooming

When zoomed out, having details like transcript structure, number of transcripts, etc can be too much information. Might allow reduced representation (e.g. just a line with arrow can work)

RCA1
------->

**Comment by irusri** [comment]
This is very useful feature for us

**Comment by cmdcolin** [comment]
thanks for posting. you are specifically interested in for example transcript text labels correct @irusri ?

**Comment by irusri** [comment]
yes you are absolutely correct @cmdcolin. So the idea is to display transcript tracks separately with own labels. Thanks!

**Comment by scottcain** [comment]
A WormBase user asked about this functionality too.
